### ISSUE TITLE
Jitter chunk sizes

### ISSUE THREAD

**Issue by brilee** [tensorflow/minigo#4]
Apparently shuffle queues will round-robin over the files it's reading. So if it's reading 64 parallel files that are exactly 10,000 examples each, then it will finish reading all 64 at exactly the same time. That could result in some issues. To get rid of that, jitter the chunk sizes to be +/-25% of the desired size.

**Comment by amj** [comment]
Given how all of our tf example reading code has its roots in a [single generator call](https://github.com/tensorflow/minigo/blob/8c6ffc7a3807614e305f188b0a27b1c5c1cea3ea/preprocessing.py#L143), i don't see a clean way to do this.  I'm also not convinced it's actually a problem at present.  It's possible we can use the parallel_interleave in tf.contrib with the `sloppy` semantics to solve it.  I'm removing this from our blockers for our 9s run and we can revisit if we have reason to believe our training data is insufficiently random.

**Comment by brilee** [comment]
I've done some independent research on the usefulness of jittering shard sizes and have concluded it's not useful.
